src/lib/y2storage/autoinst_issues/invalid_value.rb

```diff
@@ -60,11 +64,10 @@ def severity
       # @return [String] Error message
       # @see Issue#message
       def message
-        # TRANSLATORS: 'value' is a generic value (number or string)
-        # 'attr' is an AutoYaST element name; 'device' is a device name (eg. /dev/sda1);
-        # 'new_value_message' is a short explanation about what should be done with the value.
-        _("Invalid value '%{value}' for attribute '%{attr}' on '%{device}' (%{new_value_message}).") %
-          { value: value, attr: attr, device: device, new_value_message: new_value_message }
+        # TRANSLATORS: 'value' is a generic value (number or string) 'attr' is an AutoYaST element
+        # name; 'new_value_message' is a short explanation about what should be done with the value.
+        _("Invalid value '%{value}' for attribute '%{attr}' (%{new_value_message}).") %
+          { value: value, attr: attr, new_value_message: new_value_message }
```

Comment:
rubocop is fine with this? I think it usually suggest to use format, but maybe it is here more benevolent.

Reply:
Yes, it is fine.